### PR TITLE
chore: add genesis tests

### PR DESCRIPTION
**Pull request by zarazan**
## Description

- Add oracle genesis tests
- Add airdrop genesis tests
- Consolidate integration test setup

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: tests/integration/test_helpers.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	})
+
+	queryHelper := baseapp.NewQueryServerTestHelper(ctx, app.InterfaceRegistry())
+	oracletypes.RegisterQueryServer(queryHelper, oraclekeeper.NewQuerier(app.OracleKeeper))
+	airdroptypes.RegisterQueryServer(queryHelper, airdropkeeper.NewQuerier(app.AirdropKeeper))
+
+	sh := stakingtestutil.NewHelper(t, ctx, app.StakingKeeper)
+	sh.Denom = bondDenom
+
+	// amt := sdk.TokensFromConsensusPower(100, sdk.DefaultPowerReduction)
```
Comment (rbajollari) [inline tests/integration/test_helpers.go:83]: Do we still need this commented code? And where its used and also commented in line [97](https://github.com/ojo-network/ojo/blob/0551806e2ae1866cb786ff277a078e6322cc335a/tests/integration/test_helpers.go#L97)?

**Review by rbajollari — APPROVED** [review]
Just one nit. Good clean up taking out the reused code in the setup test logic🔥

**Review by adamewozniak — APPROVED** [review]
lgtm folks ❤️

File: tests/integration/test_helpers.go (hunk 1 above)
Comment (zarazan) [reply tests/integration/test_helpers.go:83]: I'll just leave this for when I cleanup the test setup in the oracle module as well. There were just a few extra issues to iron out with voting power, etc.

**Pull request merged**